.github/workflows/deploy.yaml

```
@@ -0,0 +1,80 @@
+name: Deploy to ECR
+
+on:
+  push:
+    paths:
+      - 'src/**'
+      - 'k8s/**'
+      - 'Dockerfile'
+      - 'pom.xml'
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'src/**'
+      - 'k8s/**'
+      - 'Dockerfile'
+      - 'pom.xml'
+
+
+jobs:
+  build:
+    name: Deployment
+    needs: Tests
+    runs-on: ubuntu-latest
+    env:
+      ECR_REPOSITORY: totem_express
+      EKS_CLUSTER_NAME: totem_express-8soat
```

Como a ideia é que todos os microsserviços fiquem na mesma conta da cloud e na mesma região, aqui acho melhor trocar pra algo específico desse microsserviço:
```suggestion
      ECR_REPOSITORY: totem_express_produtos_pedidos
      EKS_CLUSTER_NAME: totem_express_produtos_pedidos-8soat
```